database/migrations/2016_06_29_205018_add_oauth_fields_to_users.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
I'm not sure what this is/why we need it? I was thinking we would do Auth stuff separately since we haven't dealt with users yet. Is it related to the phoenix stuff?